lib/node/Type.js

```diff
@@ -96,5 +96,32 @@ NodeType.isIntegerFraction = function(node, allowUnaryMinus=false) {
           Number.isInteger(parseFloat(denominator.value)));
 };
 
+NodeType.isFraction = function(node, allowUnaryMinus=true, allowParens=true) {
```

Comment:
Not sure if this `isFraction` / `getFraction` pattern is okay, or if there's a better way to accomplish this? I'm sure we've run into similar situations elsewhere, not sure how it is typically handled?